<!-- %Tag(FULL)%-->
<launch>


 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="xbox_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.1" />
    <param name="autorepeat_rate" value="5" />
  </node>

 <!-- Axes -->
  <param name="X" value="4" type="int"/>
  <param name="XDir" value="1" type="int"/>
  <param name="Y" value="3" type="int"/>
  <param name="YDir" value="-1" type="int"/>
  <param name="Z" value="1" type="int"/>
  <param name="ZDir" value="1" type="int"/>
  <param name="JawRotate" value="0" type="int"/>
  <param name="JawRotateDir" value="1" type="int"/>
  <param name="JawDir" value="1" type="int"/>
  <param name="JawOpenAxis" value="2" type="int"/>
  <param name="JawCloseAxis" value="5" type="int"/>
  <param name="scale" value="0.5" type="double"/>

  <node pkg="joy_controls" type="cartesian_teleop" name="ARM5JoyCartesianControl" />

</launch>
<!-- %EndTag(FULL)%-->
